requirements.txt

```diff
@@ -1,15 +1,28 @@
+appnope==0.1.0
+decorator==4.0.9
 dj-database-url==0.3.0
 dj-static==0.0.6
-Django==1.8.5
+Django==1.9.2
 django-cities==0.4.2
 django-filter==0.11.0
 django-rest-assured==0.2.0
 djangorestframework==3.3.0
 factory-boy==2.6.0
 fake-factory==0.5.3
+gnureadline==6.3.3
 gunicorn==19.3.0
+ipdb==0.8.1
+ipython==4.1.1
+ipython-genutils==0.1.0
```

Comment:
ipdb, ipython and ipython-genutils are not requirements for the code, so they should not be in the requirements file.